open-api/constraints.md

```diff
@@ -0,0 +1,209 @@
+#Constraints
```

Comment:
if you title this document README.md then it will get automatically displayed by GitHub when someone opens this folder,